sdk/core/Azure.Core/src/Shared/OperationInternalBase.cs

```diff
@@ -15,15 +15,18 @@ namespace Azure.Core
     internal abstract class OperationInternalBase
     {
         private readonly ClientDiagnostics _diagnostics;
-        private readonly string _updateStatusScopeName;
         private readonly IReadOnlyDictionary<string, string>? _scopeAttributes;
         private readonly DelayStrategy? _fallbackStrategy;
         private readonly AsyncLockWithValue<Response> _responseLock;
 
+        protected readonly string _updateStatusScopeName;
+        private readonly string _waitForCompletionScopeName;
+
         protected OperationInternalBase(Response rawResponse)
         {
             _diagnostics = new ClientDiagnostics(ClientOptions.Default);
             _updateStatusScopeName = string.Empty;
+            _waitForCompletionScopeName = string.Empty;
```

Comment:
Curious: when do we want to open distributed tracing span with an empty name?

Reply:
Management plane is using this. It looks like for mocking purpose. The usage is like 
https://github.com/Azure/autorest.csharp/blob/0790793118f59b07bfd58047ab0dbbf1501373fe/test/TestProjects/ResourceRename/Generated/LongRunningOperation/ResourceRenameArmOperation.cs#L31
User calls this to create an operation with in a final successful/failed state.

Reply:
See comment here: https://github.com/Azure/azure-sdk-for-net/pull/29033#discussion_r890410465

Reply:
It is not for mocking purpose. This constructor is used when operation is in the final state, hence no scope will be created.